src/kit/Button.tsx

```diff
@@ -8,9 +8,12 @@ import Tooltip from 'kit/Tooltip';
 
 import css from './Button.module.scss';
 
+export type status = 'active' | 'critical' | 'inactive' | 'pending' | 'success' | 'warning';
+
 interface ButtonProps {
   block?: boolean;
   children?: ReactNode;
+  shape?: 'default' | 'circle' | 'round';
```

Comment:
nit: it's not clear from the name how `round` differs from `circle`. maybe `rounded` would be better?

Reply:
actually seemed to require a bunch of code to map to the antd prop so just going to describe it in the documentation as the "Rounded corners" variation